src/modules/fancyzones/editor/FancyZonesEditor/loc/it/src/modules/fancyzones/editor/FancyZonesEditor/Properties/Resources.resx.lcl

```diff
@@ -37,6 +37,15 @@
         </Str>
         <Disp Icon="Str" />
       </Item>
+      <Item ItemId=";Canvas_Layout_Editor" ItemType="0;.resx" PsrId="211" Leaf="true">
+        <Str Cat="Text">
+          <Val><![CDATA[Canvas layout editor]]></Val>
+          <Tgt Cat="Text" Stat="Loc" Orig="New">
+            <Val><![CDATA[Editor layout area di disegno]]></Val>
```

Comment:
I'm Italian, and this sounds very weird, it's a mix of English and Italian and kind of meaningless.